stream-compositions/api/integrations-api/payment-order-integration-api/src/main/resources/integration-api-v2.yaml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.0.3
+info:
+  title: Payment Order Integration API
+  version: 2.0.0
+servers:
+  - url: http://localhost:8080/service-api/v1
+    description: The server
+tags:
+  - name: PaymentOrderIntegration
+paths:
+  /integration-api/v2/payment-order:
+    post:
+      tags:
+        - PaymentOrderIntegration
+      summary: Pulls Payment Orders for the Arrangement and either date range of number of billing cycles.
+      description: Pulls payment orders.
+      operationId: pullPaymentOrders
+      requestBody:
+        description: Push Ingestion Request
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PullIngestionRequest'
+
+      responses:
+        "200":
+          description: Transactions sucessfully returned.
```

Comment:
I guess it should not be "Transactions" ?

Reply:
Fixed. Thanks